src/main/java/step5/domain/CarName.java

```diff
@@ -0,0 +1,21 @@
+package step5.domain;
+
+public class CarName {
+    private static final int CAR_NAME_MAX_LENGTH = 5;
+    private final String name;
```

Comment:
상수와 인스턴스 변수의 구분을 위해 개행을 추가해보면 좋을 것 같습니다.
```suggestion
    private static final int CAR_NAME_MAX_LENGTH = 5;
    
    private final String name;
```